openmetadata-service/src/main/java/org/openmetadata/service/search/elasticsearch/ElasticSearchClient.java

```diff
@@ -1032,16 +1041,18 @@ public Response searchSchemaEntityRelationship(
 
   private void getLineage(
       String fqn,
+      Set<String> visitedFqn,
```

Comment:
@mohityadav766 if a node contains 1000s of neighbors would it impact the vistitedFqn?

Reply:
should be ok @harshach , once we have the pagination bits as part of  lineage improvements it won't be a problem
and should be ok will 1000ish records for now